trade_remedies_public/templates/v2/registration/how_to_get_account.html

```diff
@@ -0,0 +1,19 @@
+{% extends "v2/govuk/base_with_form.html" %}
+
+
+{% block page_title %}
+    How to get an account
+{% endblock %}
+
+{% block heading %}
+    How to get an account
+{% endblock heading %}
+
+{% block form_content %}
```

Comment:
Then here would be row_content
```suggestion
{% block row_content %}
```